app/models/movement.rb

```diff
@@ -0,0 +1,7 @@
+class Movement < ApplicationRecord
+  validates_presence_of :movement_name, :image, :video, :description
+  
+  belongs_to :athlete
+  has_many :movement_goals, dependent: :destroy
+  has_many :progress_movements, dependent: :destroy
+end
```

Comment:
Need to add

has_many :goals, through: movement_goals
has_many :progresses, through: progress_movements